libarchive/archive_read_open_fd.c

```diff
@@ -133,13 +133,13 @@ file_skip(struct archive *a, void *client_data, int64_t request)
 	struct read_fd_data *mine = (struct read_fd_data *)client_data;
 	int64_t skip = request;
```

Comment:
`skip` here should be `off_t` since that's the type that the `lseek` calls expect.

Reply:
How? I addressed this.